patterns/cta-book-links.php

```diff
@@ -14,47 +14,47 @@
+		<div class="wp-block-button"><a class="wp-block-button__link wp-element-button"><?php echo esc_html_x( 'Amazon', 'Example brand name', 'twentytwentyfive' ); ?></a></div>
 		<!-- /wp:button -->
 
 		<!-- wp:button -->
-		<div class="wp-block-button"><a class="wp-block-button__link wp-element-button">Audible</a></div>
+		<div class="wp-block-button"><a class="wp-block-button__link wp-element-button"><?php echo esc_html_x( 'Audible', 'Example brand name', 'twentytwentyfive' ); ?></a></div>
 		<!-- /wp:button -->
 
 		<!-- wp:button -->
-		<div class="wp-block-button"><a class="wp-block-button__link wp-element-button">Barnes &amp; Noble</a></div>
+		<div class="wp-block-button"><a class="wp-block-button__link wp-element-button"><?php echo esc_html_x( 'Barnes &amp; Noble', 'Example brand name', 'twentytwentyfive' ); ?></a></div>
 		<!-- /wp:button -->
 
 		<!-- wp:button -->
-		<div class="wp-block-button"><a class="wp-block-button__link wp-element-button">Apple Books</a></div>
+		<div class="wp-block-button"><a class="wp-block-button__link wp-element-button"><?php echo esc_html_x( 'Apple Books', 'Example brand name', 'twentytwentyfive' ); ?></a></div>
 		<!-- /wp:button -->
 
 		<!-- wp:button -->
-		<div class="wp-block-button"><a class="wp-block-button__link wp-element-button">Bookshop.org</a></div>
+		<div class="wp-block-button"><a class="wp-block-button__link wp-element-button"><?php echo esc_html_x( 'Bookshop.org', 'Example brand name', 'twentytwentyfive' ); ?></a></div>
 		<!-- /wp:button -->
 
 		<!-- wp:button -->
-		<div class="wp-block-button"><a class="wp-block-button__link wp-element-button">Spotify</a></div>
+		<div class="wp-block-button"><a class="wp-block-button__link wp-element-button"><?php echo esc_html_x( 'Spotify', 'Example brand name', 'twentytwentyfive' ); ?></a></div>
 		<!-- /wp:button -->
 
 		<!-- wp:button -->
-		<div class="wp-block-button"><a class="wp-block-button__link wp-element-button">BAM!</a></div>
+		<div class="wp-block-button"><a class="wp-block-button__link wp-element-button"><?php echo esc_html_x( 'BAM!', 'Example brand name', 'twentytwentyfive' ); ?></a></div>
 		<!-- /wp:button -->
 
 		<!-- wp:button -->
-		<div class="wp-block-button"><a class="wp-block-button__link wp-element-button">Simon &amp; Schuster</a></div>
+		<div class="wp-block-button"><a class="wp-block-button__link wp-element-button"><?php echo esc_html_x( 'Simon &amp; Schuster', 'Example brand name', 'twentytwentyfive' ); ?></a></div>
 		<!-- /wp:button -->
 	</div>
 	<!-- /wp:buttons -->
 
 	<!-- wp:paragraph {"align":"center","fontSize":"medium"} -->
-	<p class="has-text-align-center has-medium-font-size">Outside Europe? View <a href="#">international editions</a>.</p>
+	<p class="has-text-align-center has-medium-font-size"><?php echo wp_kses_post( _x( 'Outside Europe? View <a href="#" rel="nofollow">international editions</a>', 'Sample text with link.', 'twentytwentyfive' ) ); ?></p>
```

Comment:
```suggestion
	<p class="has-text-align-center has-medium-font-size"><?php echo wp_kses_post( _x( 'Outside Europe? View <a href="#" rel="nofollow">international editions</a>.', 'Sample text with link.', 'twentytwentyfive' ) ); ?></p>
```

Re-add the missing period.